client/deis.py

```diff
@@ -1321,24 +1322,27 @@ def config_push(self, args):
         """
         Sets environment variables for an application.
 
-        Your environment is read from a file named .env. This file can be
-        read by foreman to load the local environment for your app.
+        The environment is read from <path>. This file can be read by foreman
+        to load the local environment for your app.
 
-        Usage: deis config:push [options]
+        Usage: deis config:push [<path>] [options]
```

Comment:
here you've added it as an argument, yet the code states it as an argument. Could you please remove the `[<path>]` here? `[options]` will populate all options